### DISCUSSION TITLE
Add delete into the notification preview along with reply and mark read.

### DISCUSSION THREAD

**Discussion by adarshkannada** [octoshrimpy/quik#172] [Ideas]
I'd like to see the ability to delete the message after reading directly from the notification preview/shade. Can this be done?

**Comment by octoshrimpy** [comment]
This functionality already exists. 

Settings > Notifications > Actions

options are:
- none
- archive
- delete
- block
- call
- mark read
- reply

and you have 3 buttons available.

**Reply by adarshkannada** [reply]
Awesome, thanks a lot.

**Reply by ADTC** [reply]
Thanks! I was looking for this, to add "Block".
